client/buildozer.spec

```
@@ -0,0 +1,97 @@
+# (str) The main .py file to use as the main entry point for your app
+source.include_exts = py,kv,png,jpg,ttf,atlas
+
+# (list) List of inclusions using pattern matching
+source.include_patterns = app.kv, services/*
+
+# (str) Application versioning (method 1)
+version = 0.1.0
+
+# (list) Application requirements
+# Ensure to keep recipes minimal for faster builds
+requirements = python3,kivy==2.3.0,kivymd,requests,kivy_garden.graph
```

RuboCop (1.76.1)

[fatal] no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)